### PR TITLE
Feat/manager 16065 poweron node

### PR DESCRIPTION
**Pull request by tibs245**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix-scale-in-out`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat #MANAGER-16065
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/modules/nutanix/src/service.js [hunk 1]
```diff
@@ -304,4 +304,18 @@ export default class NutanixService {
       ),
     };
   }
+
+  updateDedicatedNodePowerStateOn(nodeId) {
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/service.js:308]:
```suggestion
  updateClusterNodePowerStateOn(nodeId) {
```

File: packages/manager/modules/nutanix/src/service.js [hunk 2]
```diff
@@ -304,4 +304,18 @@ export default class NutanixService {
...
+    return this.$http
+      .put(`/dedicated/server/${nodeId}`, {
+        bootId: 1,
+        monitoring: false,
+        noIntervention: false,
+      })
+      .then(() => this.rebootDedicatedNode(nodeId));
+  }
+
+  rebootDedicatedNode(nodeId) {
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/service.js:318]:
```suggestion
  rebootClusterNode(nodeId) {
```

File: packages/manager/modules/nutanix/src/service.js [hunk 3]
```diff
@@ -304,4 +304,18 @@ export default class NutanixService {
...
+    return this.$http.post(`/dedicated/server/${nodeId}/reboot`);
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/service.js:319]: Could reboot be called in a standalone mode ? If not you should put it directly in 'powerStateOn' function

Comment (tibs245) [reply packages/manager/modules/nutanix/src/service.js:319]: Not but I need this same call for the poweroff function

**Review by darsene — APPROVED** [review]

**Review by vovh — APPROVED** [review]

**Pull request merged**